code/unclassified/src/lapindrom_checker/lapindrome_checker.cpp

```diff
@@ -21,8 +21,9 @@ bool isLapindrome(string str)
 
 int main()
 {
-    string test_case = "addbbadd";
- 
+    string test_case;
+    cout << "Input the string to be checked" << endl;
+    cin >> test_case;
     if (isLapindrome(test_case))
        cout<<"lapindrome\n";
```

Comment:
While you're here, you might even change the `cout` to: `cout << test_case << " is  a lapindrome.\n";`

Reply:
Sure. I'll do it right away.